.github/workflows/release.yml

```
@@ -250,8 +250,12 @@ jobs:
         run: |
           git config --global user.name "edgelessci"
           git config --global user.email "edgelessci@users.noreply.github.com"
-          git commit -m "attestation: hardcode measurements for ${VERSION}"
-          git push
+          if git status --porcelain | grep measurements_enterprise.go; then
```

```suggestion
          if git status --porcelain | grep -e measurements_enterprise.go -e image_enterprise.go; then
```
The workflow also updates the default image, so we should check that as well

---

We do this in a separate job of the workflow already